packages/components/gradientText/GradientText.tsx

```diff
@@ -123,6 +123,8 @@ const gradient = (type: GradientType): LinearGradientProps => {
       return getMapPostTextGradient(PostCategory.Normal);
     case getMapPostTextGradientType(PostCategory.Article):
       return getMapPostTextGradient(PostCategory.Article);
+    case getMapPostTextGradientType(PostCategory.ArticleMarkdown):
+      return getMapPostTextGradient(PostCategory.Article);
```

Comment:
Is this on purpose ?

Reply:
Since we have a new PosteCategory `ArticleMarkdown` (for now), yes

Reply:
I completed the description

>  Please take attention to that: At the final, we'll have only one Article category, type, view, etc. These stuff will be merged with the actual "Article" stuff, and the mention "Markdown" will be removed in the future